apps/DisplayCluster/main.cpp

```diff
@@ -76,7 +78,17 @@ int main( int argc, char* argv[] )
         return EXIT_SUCCESS;
     }
 
-    MPIChannelPtr wallChannel( new MPIChannel( *worldChannel, rank > 0, rank ));
+    const int color = rank > 0 && !isForkerProcess;
+    MPIChannelPtr localChannel( new MPIChannel( *worldChannel, color, rank ));
+    MPIChannelPtr mainChannel( new MPIChannel( *worldChannel, !isForkerProcess,
+                                               rank ));
+
+    if( isForkerProcess )
+    {
+        ProcessForker( localChannel ).run();
+        put_flog( LOG_DEBUG, "done." );
```

Comment:
The reason for it was to do the same as the master and wall processes at line 120, but I agree that it's not really useful for this simple process. There is no need to specify "forker" in the message because the put_flog function does it already.